### PR TITLE
Payee 307

### PR DESCRIPTION
**Pull request by vivekhmrc**
- [x] Unit tests passing
- [x] Coverage reached
- [ ] Acceptance tests passing
- [ ] Reviewed

### CODE REVIEW

File: test/views/html/estimatedIncomeTax/ComplexEstimatedIncomeTaxViewSpec.scala [hunk 1]
```diff
@@ -39,11 +39,17 @@ class ComplexEstimatedIncomeTaxViewSpec extends TaiViewSpec {
...
+    "have a paragraph for the Total estimated Income value " in {
+      doc(view) must haveParagraphWithText(messages("£48,000"))
     }
 
     "have a heading for the Income tax estimate" in {
-      doc(view) must haveH2HeadingWithText(messages("tai.incomeTax.incomeTaxEstimate.subheading") + " £15,000")
+      doc(view) must haveH2HeadingWithText(messages("tai.incomeTax.incomeTaxEstimate.subheading"))
+    }
+    "have a paragraph for the Income tax estimate value " in {
+      doc(view) must haveParagraphWithText(messages("£15,000"))
```
Comment (brendanstraughan) [inline test/views/html/estimatedIncomeTax/ComplexEstimatedIncomeTaxViewSpec.scala:52]: Dont think this needs to be wrapped in a messages

File: test/builders/RequestBuilder.scala [hunk 2]
```diff
@@ -50,7 +50,7 @@ object RequestBuilder {
       .withHeaders(headers.toArray: _*)
 
   def buildFakeRequestWithAuth(method: String) =
-    FakeRequest(method = method, path = "")
+    FakeRequest(method = method, path = "/")
```
Comment (brendanstraughan) [inline test/builders/RequestBuilder.scala:53]: how come this was changed?

File: app/views/includes/yourTaxGraph.scala.html [hunk 3]
```diff
@@ -66,37 +66,45 @@
...
+
                     </tr>
                 }
 
             </tbody>
         </table>
-    }
\ No newline at end of file
+    }
+
```
Comment (brendanstraughan) [inline app/views/includes/yourTaxGraph.scala.html:106]: nitpick - whitespace :)

File: app/views/estimatedIncomeTax/navigationLinks.scala.html [hunk 4]
```diff
@@ -18,29 +18,33 @@
...
 
-<div class="grid grid-1-3">
+<div class="grid grid-1-3 govuk-grid-column-one-third">
     <nav class="section">
-        <h2 class="heading-section">@Messages("tai.taxCode.sideBar.heading")</h2>
+        <h2 class="heading-section  govuk-heading-m">@Messages("tai.taxCode.sideBar.heading")</h2>
         <div class="inner-block">
-            <div class="link-list">
-                @link(
+            <ul class="link-list govuk-list govuk-list--spaced">
```
Comment (brendanstraughan) [inline app/views/estimatedIncomeTax/navigationLinks.scala.html:25]: does this being a `<ul>` make any difference

File: app/views/estimatedIncomeTax/ComplexEstimatedIncomeTaxView.scala.html [hunk 5]
```diff
@@ -38,53 +39,61 @@
...
+                    <div class="govuk-grid-row">
+                        <div class="govuk-grid-column-one-half">
+                            <h2 class="govuk-heading-m ">
                                 @Messages("tai.incomeTax.totalEstimatedIncome.subheading")
-                                <span id="total-estimated-income" class="display-block heading-48 section faded-text flush section--narrow">
-                                    @pounds(viewModel.incomeEstimate)
-                                </span>
                             </h2>
+                            <p id="total-estimated-income" class=" govuk-body govuk-!-font-size-36 govuk-!-font-weight-bold faded-text-dark">
+                            @pounds(viewModel.incomeEstimate)
```
Comment (brendanstraughan) [inline app/views/estimatedIncomeTax/ComplexEstimatedIncomeTaxView.scala.html:73]: This was a H2 before, does it still look the same?

File: app/views/estimatedIncomeTax/ComplexEstimatedIncomeTaxView.scala.html [hunk 6]
```diff
@@ -38,53 +39,61 @@
...
-                            <h2 class="flush--bottom">
+                        <div class="govuk-grid-column-one-half">
+                            <h2 class="govuk-heading-m ">
                                 @Messages("tai.incomeTax.incomeTaxEstimate.subheading")
-                                <span id="income-tax-estimate" class="display-block heading-48 section faded-text flush section--narrow">
-                                    @pounds(viewModel.incomeTaxEstimate)
-                                </span>
                             </h2>
+                            <p id="income-tax-estimate" class="govuk-body govuk-!-font-size-36 govuk-!-font-weight-bold faded-text-dark">
+                            @pounds(viewModel.incomeTaxEstimate)
```
Comment (brendanstraughan) [inline app/views/estimatedIncomeTax/ComplexEstimatedIncomeTaxView.scala.html:81]: same^

File: test/views/html/estimatedIncomeTax/ComplexEstimatedIncomeTaxViewSpec.scala (hunk 1 above)
Comment (vivekhmrc) [reply test/views/html/estimatedIncomeTax/ComplexEstimatedIncomeTaxViewSpec.scala:52]: Yes 👍

File: test/builders/RequestBuilder.scala (hunk 2 above)
Comment (vivekhmrc) [reply test/builders/RequestBuilder.scala:53]: Not entirely sure of this @brendanstraughan but this is coming from  "Is this page not working properly?"

The tai application is now calling the _contact-frontend_ service asking us to present a URL PATH “/”   whereas previously it was inbuilt so the PATH was not needed

**Pull request merged**